extensions/wikia/AffiliateService/js/ext.AffiliateService.js

```diff
@@ -117,25 +117,7 @@ require([
 		},
 
 		canDisplayUnit: function () {
-			// you're logged out?
-			if (!w.wgUserName) {
-				if (AffiliateService.isHuluCommunity()) {
-					return true;
-				}
-
-				// is that wiki whitelisted?
-				if (w.wgAffiliateEnabled) {
-					return true;
-				}
-
-				// do we have debug targeting
-				if (AffiliateService.getDebugTargeting() !== false) {
```

Comment:
Can we keep the debug param?

Reply:
https://github.com/Wikia/app/pull/17785/commits/b9c44ba0c57d198695c47ac5743b6f89619e5115